include/deal.II/lac/constraint_matrix.h

```diff
@@ -1406,6 +1421,8 @@ ConstraintMatrix::add_entry (const size_type line,
   // exists, since we don't want to enter it twice
   Assert (lines_cache[calculate_line_index(line)] != numbers::invalid_size_type,
           ExcInternalError());
+  Assert (!local_lines.size() || local_lines.is_element(column),
```

Comment:
I think the change is in general a good idea. I'd like to see the testsuite run on it. It may not be the right time for such a change a couple of days before a release, though :-)

@masterleinad : What do you think about removing this line from the patch for now and instead putting it into a separate pull request that we can postpone until after the next release?